### DISCUSSION TITLE
Any way to edit the layout in an svg editor?

### DISCUSSION THREAD

**Discussion by mirams** [slipx06/sunsynk-power-flow-card#216] [Q&A] [answered]
I have the card (which is **great**, thanks for the work on it!) showing on a raspberry pi touchscreen, so the dimensions are 800 x 480. I'm using the lite/compact card and it is 500x400, so I can have either only part of flowcard  height showing, but filling the width, or all the height showing but lots of blank space at the edges. 

I'd like to alter the svg template to use all the available space on the screen (in my own fork is fine).

So this might be a silly question, but is there a way to edit the layout in an svg editor with GUI (e.g. inkscape), or does it all need doing manually in the `index.ts` file as it is tied in with code instructions (i.e. is that how the developers alter the layout now?). I saw the earliest commits had a `draw.io` folder with the `.svg` in it, but that seems to have gone now?

**Comment by slipx06** [comment, marked as answer]
You will need to edit the index.ts file . Specifically the lines below https://github.com/slipx06/sunsynk-power-flow-card/blob/845a639c0fc6f185df90603b3812893d329b0968/src/index.ts#L3089-L3092

I would start by adjusting the preserveAspectRatio and set it to none. You can then manually set the height and width attributes.
